src/main/java/org/generationcp/middleware/api/germplasm/GermplasmService.java

```diff
@@ -82,4 +85,9 @@
 
 	void updateGermplasmPedigree(Integer gid, ProgenitorsUpdateRequestDto progenitorsUpdateRequestDto);
 
+	void mergeGermplasm(GermplasmMergeRequestDto germplasmMergeRequestDto, String crossExpansion);
+
+	List<GermplasmMergeDto> getGermplasmMergeDTOs(Integer gid);
```

Comment:
I think it's more intuitive to name as `List<GermplasmMergedDto> getGermplasmMerged(Integer gid);`